src/lineage_provider/modelGraphViewPanel.ts

```diff
@@ -93,8 +93,10 @@ export class ModelGraphViewPanel implements WebviewViewProvider {
         return;
       }
       this.g6Data = this.parseGraphData();
-      this.transmitData(this.g6Data);
-      this.updateGraphStyle();
+      if (this._panel) {
```

Comment:
this particular block is needed for the scenario:
start the plugin without lineage view active
switch to an sql model that should show lineage.
debugger log shows the error similar to thrown in issue #333